programs/server/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(CLICKHOUSE_SERVER_SOURCES
     Server.cpp
 )
 
-if (OS_LINUX AND ARCH_AMD64)
```

Comment:
BTW - that issue didn't popup on your tests https://github.com/ClickHouse/ClickHouse/issues/14002 ?

Reply:
Official build does not have that issue as we use `llvm-objcopy` instead of `objcopy`.